CMakeLists.txt

```diff
@@ -6,21 +6,25 @@
 ########################################################################
 # Project setup
 ########################################################################
-cmake_minimum_required(VERSION 2.8.12)
-project(czmq)
-enable_language(C)
+cmake_minimum_required(VERSION 3.14...3.21)
```

Comment:
breaking backward compatibility is not allowed, cmake 2.8 is still in use in supported OSes

Reply:
I know, but `fetch content` is only really working starting witch `CMake v3.14`

Please realise: **CMake v2.8 is totally historic and deprecated!**